MonoGame.Framework/Input/MessageBox.cs

```diff
@@ -0,0 +1,63 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.Xna.Framework.Input
+{
+    public partial class MessageBox
```

Comment:
Partial is allowed on static classes.  We use it already in other MonoGame code.